frappe/integrations/doctype/google_maps/google_maps.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+import datetime
+
+class GoogleMaps(Document):
+	pass
```

Comment:
where are you using the googlemaps API??

Reply:
hey @rmehta , it is used in Delivery Trip under def calculate_time_matrix(name) .